### DISCUSSION TITLE
Why mkcert is required especially for dev environment?

### DISCUSSION THREAD

**Discussion by quentincaffeino** [JaneJeon/blink#528] [Q&A] [answered]
And also why not use LetsEncrypt for prod by default?

**Comment by JaneJeon** [comment, marked as answer]
Blink does not actually manage SSL certs by itself - as I've said on the reddit thread, this is purely a HTTP application. I use mkcert for local development to allow running and testing HTTPS on localhost - to closely replicate the production environment as much as possible.

We don't "do" any SSL certs for prod. There is no "LetsEncrypt", there is no "mkcert", it simply is out of the scope of this project. All I'm recommending is to use an SSL-terminating proxy for production. How you decide to renew those certs is irrelevant.

For further reference, see here: https://docs.blink.rest/Installation/2.3%20Advanced
